src/atopile/schematic_utils.py

```diff
@@ -210,6 +223,21 @@ def get_schematic_dict(build_ctx: atopile.config.BuildContext) -> dict:
 
     return return_json
 
+def get_ato_lock_file(build_ctx: atopile.config.BuildContext) -> dict:
+    ato_lock_contents = {}
+
+    if build_ctx.project_context.lock_file_path.exists():
+        with build_ctx.project_context.lock_file_path.open("r") as lock_file:
+            ato_lock_contents = yaml.safe_load(lock_file)
+
+    return ato_lock_contents
+
+def get_pose(ato_lock_contents: dict, id: str) -> tuple[dict[str, float], float, bool, bool]:
```

Comment:
I'd ideally pull the `Pose` definition into a `viewer_core` or `viewer_common` file, so you can return the right types in all the right places

Reply:
Done